cmd/opdoc/opdoc.go

```diff
@@ -19,13 +19,12 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"io"
-	"os"
-	"strings"
-
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/data/transactions/logic"
 	"github.com/algorand/go-algorand/protocol"
+	"io"
+	"os"
+	"strings"
```

Comment:
I commented on a stray commit before, so just doing it again here.  I don't know why these standard imports got moved down.

Reply:
I have no clue either, seems like my ide did this for some reason, but I think all the imports are all the same. I'll move them back in vim and then run gofmt and make sure its not a gofmt thing